### DISCUSSION TITLE
VSCODE on windows Modern Fortran, fortls install error

### DISCUSSION THREAD

**Discussion by bynkook** [fortran-lang/fortls#357] [Q&A]
i am getting same error "Had trouble installing fortls, please install manually".  I installed fortls using "conda install fortls".  This is my settings.json
`{
    "files.trimFinalNewlines": true,
    "files.trimTrailingWhitespace": true,
    "editor.minimap.enabled": false,
    "terminal.integrated.defaultProfile.windows": "Command Prompt",
    "python.analysis.packageIndexDepths": [
        {
            "name": "sklearn",
            "depth": 2
        },
        {
            "name": "matplotlib",
            "depth": 2
        },
        {
            "name": "scipy",
            "depth": 2
        },
        {
            "name": "django",
            "depth": 2
        },
        {
            "name": "flask",
            "depth": 2
        },
        {
            "name": "fastapi",
            "depth": 2
        }
    ],
    "fortran.fortls.path": "C:\\Users\\(username)\\miniconda3\\pkgs\\fortls-2.13.0-py312h2e8e312_2\\Scripts\\",
    "fortran.linter.compilerPath": "c:\\Users\\(username)\\gcc\\bin\\gfortran.exe",
    "fortran.formatting.formatter": "fprettify",
    "fortran.formatting.path": "C:\\Users\\(username)\\miniconda3\\pkgs\\fprettify-0.3.7-pyhd8ed1ab_0\\site-packages\\fprettify",
}`

**Comment by emanspeaks** [comment]
Try adding `fortls.exe` to the end of `fortran.fortls.path`
